handbook/engineering/engineering-management.md

```diff
@@ -49,20 +49,52 @@ Here are some books that multiple people on the team have read and recommend. If
+We create temporary management positions when there is an organizational need. Some examples:
 
-In some cases, it may not be practical, or headcount planning might not allow for the Interim Manager to make a new hire. In this case, the Interim Manager and the interim manager's manager should agree on success criteria based on the role's requirements.
+- A team has grown beyond the [maximum viable team guideline](eng_org.md#maximum-viable-team), and we decide to split the team.
+- An Engineering Manager is out on extended PTO.
+- We decide to form a new team from scratch. 
 
-Once the Interim Manager's first new hire has been at Sourcegraph for 30 days or other agreed success criteria have been met, their manager will formally review the Interim Manager.
+### Acting manager
 
-This process aims to have a determination made by the interim manager's manager about the promotion to a permanent manager role after the agreed success criteria have been satisfied. Generally, we aim to have the interim period not exceed four months before turning into a full-time position.
+An Acting Manager occupies a management role temporarily and will move back to their original position after a set amount of time or other conditions. An Acting Manager could be filling in for a vacant role while we hire or providing cover for another Engineering Manager on extended PTO.
+
+### Manager in training
+
+Manager in training positions are for teammates pursuing an Engineering Manager role long term but who lack the necessary people management experience. We want to allow teammates to consider if they would like to make the role switch and verify that they would be good in the role.
+
+#### Requirements for a manager in training role
+
+The following requirements are in place to ensure we set up the manager in training for success.
+
+1. The candidate's manager or another nominated manager must commit to providing extra support to the manager in training for the duration of the trial period. They become the sponsoring manager.
+1. The candidate should have a track record of good performance at Sourcegraph and should be employed at Sourcegraph for a minimum of 6 months before taking on the manager in training role.
+1. The candidate's existing team must be in a position for them not to be involved for the whole duration of the trial period. Managers in training should not fulfill a dual role.
+1. The managed team must comply with our [minimum viable team](eng_org.md#minimum-viable-team) criteria.
+1. The managed team must have a dedicated and established Product Manager assigned to it.
+
+#### Trial period
+
+1. The trial period will last a minimum of 6 weeks and a maximum of 12 weeks.
```

Comment:
When I wrote this in the JIRA ticket, my intention was to say "we should decide on a single fixed amount for all trials and it should be between 6-12 weeks". What do you think is the right duration for a trial?

Reply:
```suggestion
1. The trial period will last 12 weeks but may be extended if agreed to by the manager in training and the sponsoring manager. The trial period should not be extended to more than 16 weeks.
```

I think 12 months is a good duration. Not too long, but long enough to get enough signal. 
There might be valid reasons to extend it, but we should cap it at 4 months.